packages/shader-lab/src/ShaderLabUtils.ts

```
@@ -27,11 +27,12 @@
     source: string,
     location: ShaderRange | ShaderPosition,
     file?: string
-  ) {
+  ): GSError {
     // #if _VERBOSE
     return new GSError(errorName, message, location, source, file);
     // #else
-    throw new Error(`[${errorName}]: ${message}`);
+    // @ts-ignore
+    return new Error(`[${errorName}]: ${message}`);
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)